.github/workflows/release.yml

```diff
@@ -23,37 +20,17 @@ env:
   RUSTC_WRAPPER: sccache
 
 jobs:
-  release:
-    runs-on: ubuntu-20.04
+  semver:
+    runs-on: ubuntu-latest
```

Comment:
This is only for the typescript. All the rust-related release actions are still using 20.04. The semver check should be okay and safe to use the latest Ubuntu.

> LGTM. So the git tag is the same as before right?

The format is a bit different. We don't need to add the "v" prefix. But all the docker image tags will stay the same as before. This only affects users who download the packages from GitHub Release Artifacts, since the URL link is slightly different.